knife/lib/chef/utils/licensing_handler.rb

```diff
@@ -30,7 +30,11 @@ def install_sh_url
 
       class << self
         def validate!
-          license_keys = ChefLicensing::LicenseKeyFetcher.fetch
+          begin
+            license_keys = ChefLicensing::LicenseKeyFetcher.fetch
+          rescue
+            return new(nil, nil)
```

Comment:
`license_keys` will be `nil` if an exception is thrown.

Reply:
Done